lib/faraday/connection.rb

```diff
@@ -464,5 +442,16 @@ def set_authorization_header(header_type, *args)
         header(*args)
       headers[Faraday::Request::Authorization::KEY] = header
     end
+
+    def find_default_proxy
+      warn 'no_proxy is unsupported' if ENV['no_proxy'] || ENV['NO_PROXY']
+      uri = ENV['http_proxy']
+      if uri && !uri.empty?
+        uri = 'http://' + uri if uri !~ /^http/i
+        return uri
+      else
+        return nil
+      end
```

Comment:
I think If the return value of a method is meaningful, `return` should be explicitly written. It's easy to read :)

But I think that the consistency of this project should be respected.
I deleted `return` dd13327